crypto/CMakeLists.txt

```diff
@@ -566,11 +566,20 @@ endif()
 if(BUILD_TESTING)
   add_subdirectory(test)
 
-  add_executable(${MEM_TEST_EXEC} mem_test.cc $<TARGET_OBJECTS:boringssl_gtest_main>)
-  target_compile_definitions(${MEM_TEST_EXEC} PRIVATE BORINGSSL_IMPLEMENTATION)
-  add_dependencies(${MEM_TEST_EXEC} global_target)
-  target_link_libraries(${MEM_TEST_EXEC} test_support_lib boringssl_gtest crypto)
-  add_dependencies(all_tests ${MEM_TEST_EXEC})
+  macro(add_test_executable executable_name test_file)
```

Comment:
Is this macro applicable to the  `RANDOM_TEST_EXEC` executable? If so, it might make it look cleaner.

Reply:
Maybe no because below part is JITTER specific.
```
if(JITTER_ENTROPY)
    target_compile_options(${RANDOM_TEST_EXEC} PUBLIC -DJITTER_ENTROPY)
  endif()
```